### DISCUSSION TITLE
Setup Permissions

### DISCUSSION THREAD

**Discussion by carlosaroca** [quasarframework/quasar#11721] [CLI - Capacitor / Cordova modes]
Hi, How you setup permissions in Quasar with Cordova Apps? Im try with a package but when I try release the app show me "cordova is not defined" so I dont know how this works. Thanks for the support

**Comment by MilosPaunovic** [comment]
Hello, please try to construct your question in a better way; this is too confusing at the moment.
Sharing a code snippet and describing a problem in depth would help you get better assistance.
